hybris-patches: common-kernel patch is now upstream

sledges opened the pull request.

Thaodan reviewed (changes requested):
Please drop the patch, see:
https://github.com/sonyxperiadev/kernel-sony-msm-4.19-common/commit/985492a4e0115e620a4c1befb4245c53fb340503

sledges committed:
hybris-patches: common-kernel patch is now upstream

Relevant PR: https://github.com/sonyxperiadev/kernel-sony-msm-4.19-common/pull/5

[hybris-patches] common-kernel patch is now upstream

Signed-off-by: Simonas Leleiva <simonas.leleiva@jolla.com>

sledges commented:
Dropped. This merge will need to be added to dhd together with new manifest https://github.com/mer-hybris/android/pull/151
